killerbee/dev_apimote.py

```diff
@@ -204,9 +204,9 @@ def pnext(self, timeout: int=100) -> Any:
         
         if self.packet_queue is None:
             packet: Optional[bytes] = None
-            start = datetime.utcnow()
+            start = datetime.now()
 
-            while packet is None:
+            while (packet is None) and ((start + timedelta(seconds=timeout)) > datetime.now()):
```

Comment:
One timeout strategy to reduce computations per round are to save:
```suggestion
            while (packet is None) and (end > datetime.now()):
```